app/javascript/components/App.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
+
+import Greeting from "./Greeting";
+
+function App() {
+    return (
+      <Router>
+        <Routes>
+          <Route path="/" element={<Greeting />} />
```

Comment:
- [ ] See what is displayed when I visit the root path. Kindly note that a greeting should be displayed instead. And it is the `root/index.html` file that should be used, not `static/index.html`.

  ![dhdy-1](https://user-images.githubusercontent.com/36542986/203064284-ef61a943-dafb-4a68-9f57-7fbf3af84ac0.jpg)

Reply:
done